Realm/Tests/PropertyTests.m

```diff
@@ -139,4 +144,18 @@ - (void)testSwiftPropertyNameValidation {
     XCTAssertNoThrow(RLMValidateSwiftPropertyName(@"__alloc"));
 }
 
+- (void)testTypeToString {
```

Comment:
1. Do we want this now that you've added `static_asserts` testing the same thing?
2. This is missing "linking objects".